docs/source/release_notes/release-notes.rst

```diff
@@ -60,25 +66,22 @@ develop
 
     *    - |devbreak|
          - The ``RateLimitingTimestampService`` in ``timestamp-impl`` has been renamed to ``RequestBatchingTimestampService``, to better reflect what the service does and avoid confusion with the ``ThreadPooledLockService`` (which performs resource-limiting).
+           Products that do not use this class directly are not affected.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/2083>`__)
 
     *    - |fixed|
-         - If sweep configs are specified in the AtlasDbConfig block, they will be ignored, but AtlasDB will no longer fail to start.
-           This effectively fixes the user break change of version ``0.47.0``.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/2134>`__)
-
-    *    - |fixed|
          - ``TransactionManager.close()`` now closes the lock service (provided it is closeable), and also shuts down the Background Sweeper.
```

Comment:
re your comment: yeah, might as well call this out as a devbreak in case anyone was doing something stupid